app/ui-utils/client/lib/avatar.js

```diff
@@ -39,5 +38,9 @@ export const updateAvatarOfUsername = function(username) {
 	$(`.sidebar-item.js-sidebar-type-d .sidebar-item__link[aria-label='${ username }'] .avatar-image`)
 		.attr('src', url);
 
+	// force reload of avatar on sidebar header
+	$(`.sidebar__header-thumb [alt='${ username }'].avatar-image`)
```

Comment:
```suggestion
	$(`.sidebar-item.js-sidebar-type-d .sidebar-item__link[aria-label='${ username }'] .avatar-image`)
		.attr('src', url);
```

This has since changed, just needs to be updated to reflect the current state of things.